linux_os/guide/system/software/gnome/dconf_use_text_backend/ansible/shared.yml

```diff
@@ -0,0 +1,26 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- name: "Force dconf to use the textfiles instead of a binary DB"
```

Comment:
Please make the name more meaningful, two tasks with same name, doing different things is very confusing.

Reply:
Good point, renamed.